src/els_code_navigation.erl

```diff
@@ -101,6 +101,6 @@ add_include_uri(#{ id := String }, Acc) ->
 beginning() ->
   #{range => #{from => {1, 1}, to => {1, 1}}}.
 
--spec filename_to_atom(string()) -> atom().
+-spec filename_to_atom(els_dt_document:id()) -> atom().
```

Comment:
How can this be valid? The `id` is defined as an `atom` and here we are expecting a `list`.

Reply:
`filename:basename/2` and `filename:extension/1` work with type [`file:name_all()`](http://erlang.org/doc/man/file.html#type-name_all) which is defined as:
```
name_all() =
    string() | atom() | deep_list() | (RawFilename :: binary())
```